tools/StaticAnalysis/Exceptions/Az.Kusto/SignatureIssues.csv

```diff
@@ -70,4 +70,7 @@
 "Az.Kusto","Remove-AzKustoScript","Remove-AzKustoScript","1","8410","Parameter ProxyUseDefaultCredentials of cmdlet Remove-AzKustoScript does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
 "Az.Kusto","Test-AzKustoScriptNameAvailability","Test-AzKustoScriptNameAvailability","1","8410","Parameter ProxyUseDefaultCredentials of cmdlet Test-AzKustoScriptNameAvailability does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
 "Az.Kusto","Update-AzKustoScript","Update-AzKustoScript","1","8100","Update-AzKustoScript Does not support ShouldProcess but the cmdlet verb Update indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
-"Az.Kusto","Update-AzKustoScript","Update-AzKustoScript","1","8410","Parameter ProxyUseDefaultCredentials of cmdlet Update-AzKustoScript does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
\ No newline at end of file
+"Az.Kusto","Update-AzKustoScript","Update-AzKustoScript","1","8410","Parameter ProxyUseDefaultCredentials of cmdlet Update-AzKustoScript does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"Az.Kusto","Invoke-AzKustoDataConnectionValidation","Invoke-AzKustoDataConnectionValidation","0","2020","The cmdlet 'Invoke-AzKustoDataConnectionValidation' no longer supports the type 'Microsoft.Azure.PowerShell.Cmdlets.Kusto.Support.EventGridDataFormat' for parameter 'DataFormat'.","Change the type for parameter 'DataFormat' back to 'Microsoft.Azure.PowerShell.Cmdlets.Kusto.Support.EventGridDataFormat'."
```

Comment:
line 74 to line 76 are not signature issues, please move to breaking change issues

Reply:
I edited this by accident, and rolled it back to the main branch on this file